packages/webapp/src/members/components/member-social-links.tsx

```diff
@@ -13,45 +16,91 @@ interface Props {
     member: MemberData;
 }
 
+const urlify = (address: string) => {
+    let domainBeginIndex = 0;
+
+    const protocolIndex = address.indexOf("//");
+    if (protocolIndex > -1 && protocolIndex <= 6) {
+        domainBeginIndex = protocolIndex + 2;
+    }
+
+    return `//${address.substring(domainBeginIndex)}`;
+};
+
 export const MemberSocialLinks = ({ member }: Props) => (
-    <nav className="space-y-2 flex flex-col">
-        <SocialButton
-            handle={member.account}
-            icon={FaBitcoin}
-            color="black"
-            href={`https://bloks.io/account/${member.account}`}
-        />
-        {member.socialHandles.telegram && (
-            <SocialButton
-                handle={member.socialHandles.telegram}
-                icon={FaTelegram}
-                color="indigo"
-                href={`https://t.me/${member.socialHandles.telegram}`}
-            />
-        )}
-        {member.socialHandles.twitter && (
+    <div className="flex flex-col sm:flex-row">
+        <div className="space-y-2">
             <SocialButton
-                handle={member.socialHandles.twitter}
-                icon={FaTwitter}
-                color="blue"
-                href={`https://twitter.com/${member.socialHandles.twitter}`}
+                handle={member.account}
+                icon={EosCommunityIcon}
+                color="black"
+                href={`${blockExplorerBaseUrl}/account/${member.account}`}
```

Comment:
As discussed, we should prob move `/account/` into the `${blockExplorerBaseUrl}` and maybe rename the variable.